data-reconciliation-app/docs/adr/05-Reporting.md

```diff
@@ -6,30 +6,30 @@ Proposed
 
 ## Context
 
-We need to build an API that reports the members/users' reconciled dataset back to them.
+We need to build an API that reports the members' and users' reconciled datasets back to them.
```

Comment:
nit: change datasets to records....as they can report on individual or multiple records